packages/now-cli/test/helpers/prepare.js

```diff
@@ -227,6 +227,11 @@ RUN echo $NONCE > /public/index.html
         },
       }),
     },
+    'failing-alias': {
+      'now.json': JSON.stringify(
```

Comment:
You need to define `index.html` so that something can actual deploy.

Also I think that the now.json should be `now.json-builds` like the other tests.